tests/020-module.test.js

```diff
@@ -0,0 +1,33 @@
+const mock = require('mock-fs');
+const serverModule = require('../');
```

Comment:
Please don't use the built version of the module (as you then loose nice line numbers in stack traces when not using things like source maps :wink:). Instead, you could, e.g., first parse the parser manually, as in `010`, and then mock the `.pegjs` file with it with something like this:

```js
const parser = require('pegjs').generate(
    fs.readFileSync('./src/grammar.pegjs', {encoding: 'utf-8'})
);
jest.mock('./grammar.pegjs', ()=>parser);
```

Reply:
Ah that's how function/module mocking works!
I didn't unterstand it first.

Reply:
Took me a bit of trial and error when doing it the first time as well. It is also different for CommonJS and ES6 imports (`require` vs. `import`), so: no worries :slightly_smiling_face: